litmus/director/tcid-ppcp01-create-cstor-pool/test.yml

```diff
@@ -66,7 +66,7 @@
             force_basic_auth: yes
             body_format: json
             body: '{"clusterId":"{{ cluster_id }}", "raidGroupConfig":{"groupDeviceCount":1, "raidType":"stripe"}}'
-            status_code: 201
+            status_code: 200
```

Comment:
can we put a range of valid status code?

Reply:
We can put multiple status codes . But I am not sure about the range of status codes @shovanmaity

Reply:
Should I add 201 also in the status code @shovanmaity

Reply:
200 – 299

Reply:
@shovanmaity I think we cannot add range of status code .But we can add two or more status code ex: `status_code: 200, 201, 203`

Reply:
Ok add 200, 201 and 202

Reply:
@shovanmaity Added 201 and 202 as status code PTAL